FF1R/Program.cs

```diff
@@ -6,10 +6,11 @@
 	using FFR.Common;
 
 	[Command(Name = "ff1r", Description = "Final Fantasy Randomizer"),
-	Subcommand("presets", typeof(Commands.Presets)),
+	 Subcommand("presets", typeof(Commands.Presets)),
 	 Subcommand("generate", typeof(Commands.Generate)),
 	 Subcommand("procgen", typeof(Commands.Procgen)),
-	 Subcommand("render", typeof(Commands.MapRender))]
+	 Subcommand("render", typeof(Commands.MapRender)),
+	 Subcommand("getdigest", typeof(Commands.GetDigest))]

```

Comment:
Again, nitpicky, but I like the sound of just "digest" more than "getdigest"

Reply:
done!